group_vars/cdhweb_prod/vars.yml

```diff
@@ -1,10 +1,13 @@
 ---
-# install root directory
-install_root: /srv/www/prod
+# Git refspec to use, either default or passed by "-e ref=<refspec>
+gitref: '{{ ref | default("main") }}'
 # email prefix for admin emails
-email_prefix: '[CDH Web] '
+email_prefix: "[CDH Web] "
 # allowed hosts for Django
 allowed_hosts:
-  - 'cdh-web.princeton.edu'
-  - 'cdh.princeton.edu'
-  - 'digitalhumanities.princeton.edu'
+  - "cdh-web.princeton.edu"
+  - "cdh.princeton.edu"
+  - "digitalhumanities.princeton.edu"
+  - "cdh-beta.princeton.edu"
```

Comment:
was this one meant to be temporary?

Reply:
github isn't doing a good job of showing me which one you mean specifically, sorry

Reply:
Sorry, I should have just put one comment and been more clear! I meant cdh-beta — was it meant as a temporary name for the migration, and now it will go away (until/unless we need it again)?

What I didn't say, and should have — this seems like a long list of names and it would be nice if we could clean some of them up. I think maybe some of that could happen at the lib-adc/nginx layer, but I'm not sure (and it's probably not urgent!).

Reply:
yes, cdh-beta was meant as temporary. I'll remove it along with the `cdh-web` one